integrations/sonarqube/CHANGELOG.md

```diff
@@ -7,6 +7,31 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes start -->
 
+## 0.1.111 (2024-11-18)
+
+
+### Improvements
+
+- Increased logs presence in integration (0.1.111)
+- Replaced calls to internal API for projects to GA version, making internal API use optional (0.1.111)
+
+### Bug Fixes
+
+- Fixed bug where issues/list API is not available for older SonarQube instance versions (0.1.105)
+
+
+## 0.1.111 (2024-11-15)
+
```

Comment:
I thought we fixed this already in earlier release. Can we update the changelog so it focuses on this project API transition and enhanced logs?